stock_picking_reorder_lines/stock.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+#
+#    Author: Alexandre Fayolle
+#    Copyright 2013 Camptocamp SA
+#
+#    Author: Damien Crier
+#    Copyright 2015 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+
+from openerp import models, api, fields
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+    _order = 'date_expected desc, sequence, id'
+
+    @api.model
+    def _get_sequence(self):
+        last_sequence = 10
```

Comment:
That's not true. Let me put you an example to see the problem:

* You add 15 lines. All lines will have sequence = 10.
* You use the widget handle to reorder one of the lines. The lines now have 1, 2, 3... 15 as sequence.
* Add a new line. The new line will appear at the bottom, but the sequence will be 10. You click on Save, and the line will be in the middle of the other lines.

This is a general problem in all the models that use a sequence. A better solution for me would be to put at least a higher number that it's difficult to reach (10000 for example), but it's still not perfect.